packages/atomic/cypress/integration/breadbox/breadbox.cypress.ts

```diff
@@ -176,36 +176,33 @@ describe('Breadbox Test Suites', () => {
     });
   });
 
-  describe('when selecting 12 facet values', () => {
-    const index = [1, 2, 0, 2, 0, 1, 2, 0, 2, 0, 1, 2];
-    function setupSelectedMulitpleFacets() {
-      setupBreadboxWithMultipleFacets();
-      FacetSelectors.showMoreButton().click();
-      cy.wait(TestFixture.interceptAliases.Search);
-      cy.wait(TestFixture.interceptAliases.UA);
-      index.forEach((i: number) => {
-        selectIdleCheckboxValueAt(FacetSelectors, i);
-        cy.wait(TestFixture.interceptAliases.Search);
-        cy.wait(TestFixture.interceptAliases.UA);
-      });
+  describe('when selecting 16 facet values', () => {
```

Comment:
That's a lot 😄

Reply:
I would have stuck with 12, but since the values are shorter, I needed a few more to get the bread box to overflow.